ui/component/claimMenuList/index.js

```diff
@@ -1,20 +1,41 @@
 import { connect } from 'react-redux';
-import { makeSelectClaimForUri, makeSelectClaimIsMine } from 'lbry-redux';
+import { makeSelectClaimForUri, selectMyActiveClaims, makeSelectFileInfoForUri, doPrepareEdit } from 'lbry-redux';
```

Comment:
I'm not sure we can use selectMyActiveClaims for this feature - this would require knowing all your claims, which we don't run anywhere on the web at least since it's an expensive operation, and could be lots of claims to store in redux. 

We should go back to using the "claim is mine" - this comes from the resolve/claim search result: ```      "is_my_output": true,
```

True means it's their claim.   This should work with both channels and claims.

Reply:
yeah that makes sense, don't know why I thought it would be better